lib/connect/api.js

```diff
@@ -136,7 +136,6 @@ let withUserConnections = exports.withUserConnections = co.wrap(function * (cont
 exports.withConnection = function (context, heroku) {
   return co(function * () {
     let connections = yield withUserConnections(context, context.app, context.flags, true, heroku)
-    console.log('in withConnection:', connections)
```

Comment:
Oops. Thanks for catching this. I must have forgotten this when I was debugging something. :tada: